src/dxso/dxso_compiler.cpp

```diff
@@ -762,6 +763,10 @@ namespace dxvk {
         // We could also be depth compared!
         DclSampler(idx, binding, samplerType, true, implicit);
       }
+
+      const uint32_t offset = idx * 2;
+      uint32_t textureBits = uint32_t(viewType - VK_IMAGE_VIEW_TYPE_1D);
```

Comment:
subtracting `VK_IMAGE_VIEW_TYPE_1D` (i.e. 0) here seems pointless? We're just encoding the enum as-is anyway.